templates/api/knowledge-panels/environment/ecoscore/ecoscore_extended.tt.json

```diff
@@ -0,0 +1,95 @@
+[% SET climate_change_ratio_to_category = panel.climate_change / panel.ecoscore_extended_data_for_category.climate_change %]
+
+
+    // "title": "[% f_lang('f_equal_to_driving_km_in_a_petrol_car',  { 'kilometers' => driving_100g_rounded } ) %]",
```

Comment:
?

Reply:
This is just a comment to show how to use the f_lang function, which we'll need to do at some point. In this PR I just hardcoded English strings everywhere as it's just experimental.

Reply:
ok, I understood you were hardcoding english and it's fine for an experimental panel.